apps/tup-cms/src/taccsite_cms/secrets.sample.py

```diff
@@ -1,4 +1 @@
 TUP_SERVICES_ADMIN_JWT = "CHANGEME"
-
-EMAIL_BACKEND = 'django.core.mail.backends.smtp.EmailBackend'
```

Comment:
**Note for Posterity.** [Core-CMS allows e-mail to still be testable locally by using `
EMAIL_BACKEND = 'django.core.mail.backends.console.EmailBackend'` by default.](https://github.com/TACC/Core-CMS/blob/v4.9.0/taccsite_cms/_settings/email.py#L22)